monero-outreach-docs/translations/fr/storyboard/wallet_quickstart_fr.md

```diff
@@ -1,6 +1,6 @@
-# Portefueille Monero, démarrage rapide
-*22/04/2019*  
-Almutasim  
+# Portefeuille Monero, démarrage rapide
+*22/04/2019* Almutasim  
```

Comment:
Please keep the original format of the document.

Reply:
Yes please keep the original format of the document.